Updating article to add WebSuggestedReplies (#1914)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/organization/Set-OrganizationConfig.md

Accept wildcard characters: False
```

### -WebSuggestedRepliesDisabled
The WebSuggestedRepliesDisabled parameter specifies whether to enable or disable Suggested Replies in Outlook on the web.  This feature provides suggested replies to emails so users can easily and quickly respond to messages. Valid values are:

- $true: Suggested Replies are disabled. 

> - $false: Suggested Replies are disabled. This is the default value.

Reviewer: ```suggestion
- $false: Suggested Replies are enabled. This is the default value.
```

😇

Accept wildcard characters: False
```

### -WebSuggestedRepliesDisabled

> The WebSuggestedRepliesDisabled parameter specifies whether to enable or disable Suggested Replies in Outlook on the web.  This feature provides suggested replies to emails to easily and quickly respond to messages. Valid values are:

Reviewer: ```suggestion
The WebSuggestedRepliesDisabled parameter specifies whether to enable or disable Suggested Replies in Outlook on the web.  This feature provides suggested replies to emails so users can easily and quickly respond to messages. Valid values are:
```

- $true: Suggested Replies are disabled. 

> - $false: Connectors are disabled. This is the default value.

Reviewer: ```suggestion
- $false: Suggested Replies are enabled. This is the default value.
```
